ui/components/ui/review-spending-cap/index.scss

```diff
@@ -0,0 +1,33 @@
+.review-spending-cap {
+  &__heading {
+    width: 100%;
+  }
+
+  &__heading-title {
+    &__tooltip {
+      width: 180px;
+
+      &__warning-icon {
+        color: var(--color-warning-default);
+      }
+
+      &__question-icon {
+        color: var(--color-icon-muted);
+      }
+    }
+  }
+
+  &__heading-detail {
+    flex-grow: 1;
+    align-self: center;
```

Comment:
@georgewrmarshall marking this as a request for extending Box props.

Reply:
Noted. https://github.com/MetaMask/metamask-extension/issues/15685